packages/acceptance-tests/pkg-tests-core/sources/utils/makeTemporaryEnv.ts

```diff
@@ -1,15 +1,15 @@
-/* @flow */
+import {NodeFS}    from '@yarnpkg/fslib';
+import {delimiter} from 'path';
 
-const {delimiter} = require(`path`);
-const isWsl = require(`is-wsl`);
-const {URL} = require(`url`);
+import {URL}       from 'url';
 
-const {
-  tests: {generatePkgDriver, startPackageServer, getPackageRegistry},
-  exec: {execFile},
-  fs: {createTemporaryFolder},
-} = require(`pkg-tests-core`);
-const {NodeFS} = require('@yarnpkg/fslib');
+import * as exec   from './exec';
+import * as fs     from './fs';
+import * as tests  from './tests';
+
+const {generatePkgDriver} = tests;
+const {execFile} = exec;
+const {createTemporaryFolder} = fs;
 
 global.makeTemporaryEnv = generatePkgDriver({
```

Comment:
To give some context, the initial idea was that by abstracting `makeTemporaryEnv` in a "setup file", it became possible to not only run the testsuite on Yarn, but also other package managers as well (for the tests that make sense, of course, mainly `basic`). Other package managers didn't express interest in this, so we only have the Yarn setup 🙂